frontend-react/package.json

```diff
@@ -190,10 +190,6 @@
         "vite-plugin-svgr": "^4.2.0",
         "vitest": "^1.6.0"
     },
-    "resolutions": {
-        "@types/react": "18.3.3",
```

Comment:
18.3.3 is the current default (see yarn.lock).  Is there a reason to hold it there for the future?

Reply:
It's to prevent drift with transient dependencies which wreaks havoc with typescript. It's stable right now but if the react typings code owners make updates with minor type changes it can cause builds to fail.

Reply:
Also, @types dependencies are pure types and not code. There are no security concerns with these dependencies.